.golangci.yml

```diff
@@ -1,109 +1,28 @@
+version: "2"
 linters:
-  disable-all: true
+  default: none
   enable:
-#   - asasalint
-#   - asciicheck
-#   - bidichk
-#   - bodyclose
-#   - containedctx
-#   - contextcheck
-#   - cyclop
-#   - deadcode
-#   - decorder
-#   - depguard
-#   - dogsled
-#   - dupl
-#   - dupword
-#   - durationcheck
```

Comment:
i wonder if we should review what all these linters throw up, just in case some of them are useful?